doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.md

```diff
@@ -316,3 +320,52 @@ for container environment variables.
 ## Next steps
 
 See [(Advanced) Understanding the Ray Autoscaler in the Context of Kubernetes](ray-k8s-autoscaler-comparison) for more details about the relationship between the Ray Autoscaler and Kubernetes autoscalers.
+
+(kuberay-autoscaler-v2)=
+### Autosclaer V2 with KubeRay
+
+#### Prerequisites
+* Ray 2.10.0 or nightly ray version
+* KubeRay 1.1.0 or later
+
+Ray 2.10.0 introduces Ray Autoscaler V2 alpha with KubeRay, which has improvements on observability and stability:
+1. **Observability**: The Autoscaler V2 provides instance level tracing on each ray worker node's lifecycle, making it easier to debug and understand the Autoscaler's behavior. It also reports 
+the idle information (why it's idle, why it's not idle) of each node.
+2. **Stability**: The Autoscaler V2 has improvements on idle node termination. In current V1 autoscaler, it might kill a node which was idle, but no longer idle when the termination request was processed, causing tasks or actors to fail. Leveraging on ray's graceful draining mechanism, the Autoscaler V2 can terminate idle nodes more gracefully such that no tasks or actors on an idle node will be killed.
```

Comment:
```suggestion
2. **Stability**: The Autoscaler V2 has improvements on idle node termination. In current V1 autoscaler, it might kill a node which was idle, but no longer idle when the termination request was processed, causing tasks or actors to fail. Leveraging on Ray's graceful draining mechanism, the Autoscaler V2 can terminate idle nodes more gracefully such that no tasks or actors on an idle node will be killed.
```